src/order/lattice.lean

```diff
@@ -134,6 +134,9 @@ le_antisymm
 
 instance sup_is_associative : is_associative α (⊔) := ⟨@sup_assoc _ _⟩
 
+lemma sup_left_right_swap (a b c : α) : a ⊔ b ⊔ c = c ⊔ b ⊔ a :=
+by rw [sup_comm, @sup_comm _ _ a, sup_assoc]
```

Comment:
Do we even have this lemma for addition / multiplication?

Reply:
Probably not, but we can use tactics that help us manipulate those expressions. I found myself rewriting with `inf_left_right_swap` several times in proving the associativity of `symm_diff`, and I figured I'd add the `sup` version as well.

Is there an easy way to use `ring` or `abel` with `sup` and `inf`?

Reply:
Fair enough. It looks like `sup_comm` has implicit arguments that should be explicit, but I guess that's something for another PR.